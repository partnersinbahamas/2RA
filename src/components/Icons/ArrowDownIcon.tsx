import React from 'react';
import PropTypes from 'prop-types';
import Svg from './Svg';

type TProps = {
  className?: string;
  color?: string;
  style?: { [cssAttribute: string]: string };
};

/**
 * @deprecated Use ChevronDownIcon instead.
 */
const ArrowDownIcon: React.FC<TProps> = ({ className, color, style }) => (
  <Svg
    xmlns="http://www.w3.org/2000/svg"
    viewBox="5 9 22 15"
    style={style}
    fill={color}
    className={className}
  >
    <g>
      <polygon points="5,13 8,10 16,18 24,10 27,13 16,24" />
    </g>
  </Svg>
);

ArrowDownIcon.propTypes = {
  className: PropTypes.string,
  color: PropTypes.string,
  style: PropTypes.any,
};

ArrowDownIcon.defaultProps = {
  className: '',
  color: 'currentcolor',
  style: {},
};

export default ArrowDownIcon;
