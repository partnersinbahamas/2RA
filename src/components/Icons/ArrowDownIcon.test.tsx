import React from 'react';
import { render } from '@testing-library/react';
import ArrowDownIcon from './ArrowDownIcon';

test('ArrowDownIcon', () => {
  const { container } = render(<ArrowDownIcon />);
  expect(container).toMatchSnapshot();
});
