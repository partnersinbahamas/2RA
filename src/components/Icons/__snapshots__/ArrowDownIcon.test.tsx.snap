// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArrowDownIcon 1`] = `
<div>
  <svg
    class="sc-beySPh jroDkm"
    fill="currentcolor"
    viewBox="5 9 22 15"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g>
      <polygon
        points="5,13 8,10 16,18 24,10 27,13 16,24"
      />
    </g>
  </svg>
</div>
`;
